webapp/src/main/java/com/box/l10n/mojito/pagerduty/PagerDutyIntegrations.java

```diff
@@ -0,0 +1,41 @@
+package com.box.l10n.mojito.pagerduty;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.stereotype.Component;
+
+@Component
+@DependsOn("pagerDutyIntegrationsConfiguration")
+public class PagerDutyIntegrations {
+
+  private final PagerDutyIntegrationsConfiguration pagerDutyIntegrationsConfiguration;
+
+  private Map<String, PagerDutyClient> pagerDutyClients;
+
+  @Autowired
+  public PagerDutyIntegrations(
+      PagerDutyIntegrationsConfiguration pagerDutyIntegrationsConfiguration) {
+    this.pagerDutyIntegrationsConfiguration = pagerDutyIntegrationsConfiguration;
+
+    setPagerDutyClients();
+  }
+
+  public Optional<PagerDutyClient> getPagerDutyClient(String integration) {
+    return pagerDutyClients.containsKey(integration)
+        ? Optional.of(pagerDutyClients.get(integration))
+        : Optional.empty();
+  }
+
+  public Optional<PagerDutyClient> getDefaultPagerDutyClient() {
+    return getPagerDutyClient("default");
+  }
+
+  public void setPagerDutyClients() {
```

Comment:
I don't think this method name is appropriate. This does not set anything. 
Conventionally `setX()` takes a parameter and sets an class member to x. This feels more like an init method